internal/controller/api/v1alpha1/cluster_types.go

```diff
@@ -31,18 +31,18 @@ type ClusterSpec struct {
 	// Installation will start once cluster is in known state and approved
 	Approved                 bool   `json:"approved,omitempty"`
 	OpenshiftVersion         string `json:"openshiftVersion"`
-	BaseDnsDomain            string `json:"baseDnsDomain,omitempty"`
-	ClusterNetworkCidr       string `json:"clusterNetworkCidr,omitempty"`
+	BaseDNSDomain            string `json:"baseDNSDomain,omitempty"`
+	ClusterNetworkCIDR       string `json:"clusterNetworkCIDR,omitempty"`
 	ClusterNetworkHostPrefix int64  `json:"clusterNetworkHostPrefix,omitempty"`
-	ServiceNetworkCidr       string `json:"serviceNetworkCidr,omitempty"`
-	ApiVip                   string `json:"apiVip,omitempty"`
-	ApiVipDnsName            string `json:"apiVipDnsName,omitempty"`
-	IngressVip               string `json:"ingresVip,omitempty"`
-	MachineNetworkCidr       string `json:"machineNetworkCidr,omitempty"`
-	SshPublicKey             string `json:"sshPublicKey,omitempty"`
-	VipDhcpAllocation        bool   `json:"vipDhcpAllocation,omitempty"`
-	HttpProxy                string `json:"httpProxy,omitempty"`
-	HttpsProxy               string `json:"httpsProxy,omitempty"`
+	ServiceNetworkCIDR       string `json:"serviceNetworkCIDR,omitempty"`
+	APIVIP                   string `json:"APIVIP,omitempty"`
+	APIVIPDNSName            string `json:"APIVIPDNSName,omitempty"`
+	IngressVIP               string `json:"ingressVIP,omitempty"`
+	MachineNetworkCIDR       string `json:"machineNetworkCIDR,omitempty"`
+	SSHPublicKey             string `json:"SSHPublicKey,omitempty"`
+	VIPDhcpAllocation        bool   `json:"VIPDhcpAllocation,omitempty"`
```

Comment:
DHCP needs to be all-caps in both forms.

Reply:
Ack